pinot-common/src/main/java/org/apache/pinot/common/metrics/BrokerGauge.java

```diff
@@ -42,6 +42,12 @@ public enum BrokerGauge implements AbstractMetrics.Gauge {
   NETTY_POOLED_ARENAS_DIRECT("arenas", true),
   NETTY_POOLED_ARENAS_HEAP("arenas", true),
 
+  /**
+   * The type of Adaptive Server Selector.
+   * See {@link org.apache.pinot.spi.utils.CommonConstants.Broker.AdaptiveServerSelector.Type}
+   */
+  ADAPTIVE_SERVER_SELECTOR_TYPE("adaptiveServerSelectorType", true),
```

Comment:
Can you add this at the end of all gauges?

Currently it is sitting between netty gauges.

Reply:
Thanks Ankit, addressed